hack/markdownlint.sh

```diff
@@ -8,8 +8,7 @@ CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 if [ "${IS_CONTAINER}" != "false" ]; then
   TOP_DIR="${1:-.}"
   find "${TOP_DIR}" \
-       \( -path ./vendor -prune -o \
-       -path ./tools/operator-sdk -prune \) \
+       \( -path ./vendor -prune -o \) \
```

Comment:
I don't think we need the `-o` there?
```suggestion
       \( -path ./vendor -prune \) \
```